setup.py

```diff
@@ -7,8 +7,7 @@
     readme = readme_file.read()
 
 requirements = [
-    "six",
-    "wrapt",
+    "six==1.14.0" "wrapt",
```

Comment:
missing comma

Reply:
also looks like you have removed all uses of six so happy to just drop that dep entirely

Reply:
there appears to be a dependency mismatch in the installed packages -- see https://travis-ci.org/github/square/blocks/jobs/734451161. my hope was that pinning a specific six version would allow that import to work, though I prefer also dropping six altogether.